test/vanilla/version-tolerant/Expected/AcceptanceTests/BodyFormDataVersionTolerant/bodyformdataversiontolerant/aio/operations/__init__.py

```diff
@@ -12,6 +12,24 @@
 from ._patch import *  # type: ignore # pylint: disable=unused-wildcard-import
 from ._patch import patch_sdk as _patch_sdk
 
+
+def raise_not_implemented(target):
+    _all_abstract_methods = [
+        "FormdataOperations.upload_file",
+        "FormdataOperations.upload_files",
+    ]
+    raise NotImplementedError(
+        f"{target} Not Implemented! "
+        f"We won't initialize if any of these methods not Implemented by customization: {_all_abstract_methods} "
+        "Please refer to https://aka.ms/azsdk/python/dpcodegen/python/customize to learn how to customize."
+    )
+
+
+if not callable(getattr(FormdataOperations, "upload_file", None)):
+    raise_not_implemented("FormdataOperations.upload_file")
+if not callable(getattr(FormdataOperations, "upload_files", None)):
+    raise_not_implemented("FormdataOperations.upload_files")
+
```

Comment:
Also the raise_not_implemented should be in the `__init__` of the definition of the operation group, not in the `__init__` file. Can you make sure in your testing that with our new checks, if users try to initialize a client that has an operation group with unimplmeneted methods, that they get a `NotImplementedError`?

Reply:
Good catching! Being in `__init__` of operation group should be more consistent with `abstract` design in abc.
Yes, I verified locally that user will get a `NotImplementedError` when they import the client with code in current PR.

Let me move logic to `__init__` of operation group, and then it should be raised while initializing the client.